### ISSUE TITLE
In flight query deduplication does not work

### ISSUE THREAD

**Issue by chughes87** [apollographql/apollo-client#10131]
**Intended outcome:**
Running the same queries on n sibling nodes results in n identical queries being sent over the wire rather than one.

**Actual outcome:**
N queries are sent over the wire.

**How to reproduce the issue:**
Run the same queries on n sibling nodes

**Versions**
npx: installed 1 in 1.314s

  System:
    OS: Linux 5.15 Ubuntu 20.04.5 LTS (Focal Fossa)
  Binaries:
    Node: 14.17.2 - ~/.nodenv/versions/14.17.2/bin/node
    Yarn: 3.0.1 - ~/.nodenv/versions/14.17.2/bin/yarn
    npm: 6.14.13 - ~/.nodenv/versions/14.17.2/bin/npm
  Browsers:
    Chrome: 94.0.4606.71
  npmPackages:
    @apollo/client: ^3.5.5 => 3.5.10
    apollo-boost: ^0.4.7 => 0.4.7
    apollo-upload-client: ^13.0.0 => 13.0.0

**Comment by chughes87** [comment]
Same issue as https://github.com/apollographql/apollo-client/issues/956. Opening this so that there's more visibility into the issue.

**Comment by chughes87** [comment]
I was emboldened by other's claims on the root issue. However, after much debugging, I found that the issue is with my app's apolloWrapper method! The method wasn't made such that it would create one and only one apollo client instance. I am wrapping several different methods because we are in our infancy of transitioning from a traditional Rails app to a SPA frontend with a Rails backend. So we have "islands" of React for now. I've adjusted the wrapper and in-flight deduplication now works! Sorry for the noise.
